resources/builder.inc.sh

```diff
@@ -808,11 +838,10 @@ _builder_parameter_error() {
   local program="$1"
   local type="$2"
   local param="$3"
-  echo "$COLOR_RED$program: invalid $type: $param$COLOR_RESET"
+  builder_echo red "$program: invalid $type: $param"
```

Comment:
Generally, is `builder_die` used for `builder_echo red`?

Reply:
It's possible to have a script emit an error before failing. But generally, yes, `builder_die` is the most common way we would want to emit red 'error' text.